app/components/works/show/authors_contributors_component.html.erb

```diff
@@ -0,0 +1,20 @@
+  <table id="<%= participant_type %>s" class="table table-sm caption-header">
+    <caption><%= participant_type.titlecase %>
+      <%= helpers.turbo_frame_tag dom_id(work, 'edit_author'), src: edit_button_work_path(work, tag: 'author'), target: '_top' %>
+    </caption>
+    <tbody>
+    <% participants.each do |participant| %>
+      <tr>
+        <% if participant.person? %>
+          <td style="width: 50%">
+            <%= participant.first_name %> <%= participant.last_name %>
+          </td>
+        <% else %>
+          <td width="50%"><%= participant.full_name %></td>
+        <% end %>
+        <td style="width: 25%"><%= render Works::Show::OrcidComponent.new(orcid: participant.orcid) %></td>
+        <td style="width: 25%"><%= participant.role %></td>
+      </tr>
```

Comment:
made columns fixed with or else they may line up the same between the two tables (e.g the "contributor type" column will align differently between the two tables)